### PR TITLE
preflist support

### PR DESCRIPTION
**Pull request by christophermancini**

### CODE REVIEW

**Comment by christophermancini** [comment]
Run tests with `./vendor/bin/phpunit PreflistTest tests/functional/PreflistTest.php`

**Comment by debo** [comment]
Hahahahaha @christophermancini `remove var_dump... the only way to debug php!` that's a joke right?

**Comment by christophermancini** [comment]
@debo your right... sometimes I use print_r($var, true) too! :stuck_out_tongue:

**Comment by debo** [comment]
oh come on... you are taking the p..s now :smile:

**Comment by christophermancini** [comment]
LOL, nah. This is actually super old code that I had to dust off because the preflist end point on the HTTP interface was finally fixed on Riak. I finally learned how to use xdebug / zend_debug and don't use var_dump much anymore but sometimes its just faster. :wink:

**Comment by debo** [comment]
True that. Also, once more, thanks for the great job you are doing.

File: tests/functional/PreflistTest.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    {
+        // build an object
+        $location = (new Command\Builder\StoreObject($riak))
+            ->buildObject('some_data')
+            ->buildBucket('users')
+            ->build()
+            ->execute()
+            ->getLocation();
+
+        // build an object
```
Comment (lukebakken) [inline tests/functional/PreflistTest.php]: `// build a fetch command`

File: tests/functional/PreflistTest.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        $command = (new Command\Builder\FetchPreflist($riak))
+            ->atLocation($location)
+            ->build();
+
+        $response = $command->execute();
+
+        $this->assertEquals('200', $response->getStatusCode(), $response->getBody());
```
Comment (lukebakken) [inline tests/functional/PreflistTest.php]: Check the response that it contains at least one item for the preflist?

**Comment by lukebakken** [comment]
:shipit:

**Pull request merged**